### PR TITLE
[jvm-packages] Added latest version number example

### PR DESCRIPTION
**Pull request by jamesliu**
@CodingCat 
Added latest version number example to make it easier to follow the instruction to add xgboost jvm packages.

### CODE REVIEW

File: jvm-packages/README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ if you want to use `xgboost4j-spark`, you just need to replace xgboost4j with `x
 
 ## Examples
 
+latest_version_num=0.72
```
Comment (CodingCat) [inline jvm-packages/README.md]: would you mind also modifying the doc by adding the instructions to refer the dependency? https://github.com/dmlc/xgboost/blob/master/doc/jvm/index.md

Comment (jamesliu) [reply jvm-packages/README.md]: Sure. I will do.

File: doc/jvm/index.md [hunk 2]
```diff
@@ -23,13 +23,22 @@ Or, if you would like to skip tests, run
...
     </dependency>
 
+For sbt, please add the repository and dependency in build.sbt as following:
+
+<b>sbt</b> 
+```sbt
+resolvers += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"
 
+"ml.dmlc" % "xgboost4j" % "0.72"
+```
```
Comment (CodingCat) [inline doc/jvm/index.md]: I was actually talking about how to add github snapshot repo

**Comment by jamesliu** [comment]
@CodingCat 
Added installation from repo as you suggested.

File: jvm-packages/README.md (hunk 1 above)
Comment (CodingCat) [reply jvm-packages/README.md]: in all places containing hard-coded version number, we might want to add note like "please check [Here](https://github.com/dmlc/xgboost/releases) for the latest release number"

**Comment by CodingCat** [comment]
thanks, LGTM, will wait for CI to merge

**Pull request merged**